packages/ui/src/composables/useDomRect.ts

```diff
@@ -24,6 +24,14 @@ export const useDomRect = (target: Ref<HTMLElement | undefined>) => {
     prev = rect
   })
 
+  watch(target, (newVal) => {
```

Comment:
This thing is important. It will update domRect exactly after it appears in DOM.